src/encoded/schemas/analysis.json

```diff
@@ -0,0 +1,110 @@
+{
+    "title": "Analysis",
+    "description": "Schema for grouping files into one analysis.",
+    "comment": "An analysis is typically performed on one experiments and is initially designed to group only files from a single experiment.",
+    "id": "/profiles/analysis.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["files"],
```

Comment:
also, future considerations, I suspect this will not be required.  If we post ENCAN at pipeline start time.